core-java-modules/core-java-14/src/test/java/com/baeldung/java14/foreign/api/ForeignMemoryUnitTest.java

```diff
@@ -0,0 +1,84 @@
+package com.baeldung.java14.foreign.api;
+
+import jdk.incubator.foreign.*;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.BlockJUnit4ClassRunner;
+
+import java.lang.invoke.VarHandle;
+import java.nio.ByteOrder;
+
+import static org.hamcrest.core.Is.*;
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.*;
+import static org.junit.Assert.assertThat;
+
+public class ForeignMemoryUnitTest {
+
+    @Test
+    public void whenAValueIsSet_thenAccessTheValue() {
+        long value = 10;
+        MemoryAddress memoryAddress = MemorySegment
+                .allocateNative(8).baseAddress();
```

Comment:
FWIW, I find this harder to read than if it were:

```java
        MemoryAddress memoryAddress = 
            MemorySegment.allocateNative(8)
                .baseAddress();
```

The reason is that the hanging `MemorySegment` on the original, is lost from my memory when I read the following line

Reply:
I have aligned the code snippets as per your suggestion.